data/json/professions.json

```diff
@@ -1696,7 +1696,7 @@
     "type": "profession",
     "ident": "patient",
     "name": "Patient",
-    "description": "When the diagnosis came back positive, you were willing to fight to keep living.  Now, you must renew your vow of tenacity in these new times.",
+    "description": "When the diagnosis came back positive, you made a vow: to fight for your life, and to never give in to despair.",
```

Comment:
I think this could be improved by keeping some sort of line about renewing the vow.